intro.js

```diff
@@ -262,7 +262,11 @@
       }
       var temp = {};
       for (var key in object) {
-          temp[key] = _cloneObject(object[key]);
+          if (object[key] instanceof jQuery) {
```

Comment:
Here you need to check if the jQuery is defined. I think this line can throw an error.